### DISCUSSION TITLE
[request] Webhooks

### DISCUSSION THREAD

**Discussion by sonicaghi** [Kong/kong#7357] [Ideas and feature requests]
Send notifications for specific triggered events like:
- Reached rate limit
- Authentication failed
- New API key
- etc

**Comment by lucamaraschi** [comment]
:+1:

**Comment by sonicaghi** [comment]
Scenarios

trigger emails (in 3rd party system) based on plugin actions ratelimit / quota / billing
trigger actions in API provider system / CRM based on events such as new consumer, update keys, etc ...

Behavior

a plugin or perhaps a core system API endpoint
custom callback url (anywhere on the network)
send JSON data matching the action / plugin related details

**Comment by rupzzx** [comment]
Which plugins can be used for triggering email in KONG.
